### PR TITLE
test(init.js): to reach 100% code coverage.

### LINKED ISSUE #79: Bring code coverage to 100% (again)
Since https://github.com/hoodiehq/hoodie-client/issues/46 we had some refactoring and bugfixing, and the code coverage fell down to 91% (that was me, I’m sorry :( )

current status: [Coverage Status](https://coveralls.io/r/hoodiehq/hoodie-client?branch=master)

```
$ npm run test:coverage

=============================== Coverage summary ===============================
Statements   : 91.61% ( 131/143 )
Branches     : 87.18% ( 34/39 )
Functions    : 84.38% ( 27/32 )
Lines        : 91.61% ( 131/143 )
================================================================================
```

After you run `npm run test:coverage` you can open `coverage/lcov-report/index.html` in your browser, it will show a lot of details on which parts of the code is not yet covered. There are a few lines not covered in client/lib/init.js, here’s a preview

curent-coverage

---

- [x] claim this issue (@mccullagh)
- [x] [Setup repository locally](https://github.com/hoodiehq/hoodie-client#testing) and make sure
      that all tests pass
- [x] Start a Pull Request #85
- [x] write tests until 100% coverage is reached. Check current coverage by running `npm run test:coverage`
- [ ] review & merge

Ping us in the [Hoodie Chat](http://hood.ie/chat/) or on [Twitter](https://twitter.com/hoodiehq) if you have ​​_any_​​ questions

### PR DESCRIPTION
**Pull request by mccullagh**
closes #79

### CODE REVIEW

**Comment by gr2m** [comment]
Wow, that is spot on! Thanks for the great work! Is this your first PR to an open source project?

LGTM. PIng @hoodiehq/maintainers

**Comment by mccullagh** [comment]
yes - and i realized that there are some spelling mistakes ....

File: tests/specs/init.js [hunk 1]
```diff
@@ -197,3 +197,158 @@ test('hoodie.store initialization without session', function (t) {
...
+test('"hoodie.store.*" is *not* called when "hoodie.account.isSignedIn()" returns "false"', function (t) {
+  t.plan(1)
+
+  var hoodie = {
+    account: {
+      id: 0,
+      on: simple.stub(),
+      isSignedIn: simple.stub().returnWith(false)
+    },
+    store: simple.stub(),
```
Comment (gr2m) [inline tests/specs/init.js]: the method we want to stub here is not `hoodie.store()` but `hoodie.store.connect()`. Sorry I missed this before, could you please change it?

**Comment by gr2m** [comment]
> i realized that there are some spelling mistakes

In the PR? Then best to fix them right away. If you found typos in other places it’d be best start separate pull requests

**Comment by mccullagh** [comment]
Do i need to do something?

**Comment by gr2m** [comment]
@mccullagh no you are all good, thanks. Usually we need two people from the maintainers team to comment with "LGTM", only then then `approvals/lgtm` status will be green.

But as many of our maintainers are currently conferencing through Europe and this PR doesn’t touch actual code, I’ll take advantage of my admin rights and will just merge it in :) Thanks for the great work!

**Pull request merged**